includes/theme.php

```diff
@@ -289,7 +289,16 @@ function ap_post_actions( $_post = null ) {
 		$actions[] = array( 'header' => true );
 	}
 
-	if ( ap_user_can_delete_post( $_post->ID ) ) {
+	$answers = get_posts(
```

Comment:
@mebishalnapit Why you need to load all posts here? Can't we simply get a count?

Reply:
@rahularyan `wp_count_posts` will not work on this condition since it will count all of the available answers throughout the site while in this case, we'll only need the answers related to a specific question.

Reply:
Try this: `ap_count_published_answers`

Reply:
Done and also resolved conflicts.